build.gradle

```diff
@@ -176,7 +176,7 @@ dependencies {
     // Dependency change for including MLLib
     compile('com.esotericsoftware:reflectasm:1.10.0:shaded')
 
-    compile('com.intel.gkl:gkl:0.4.1') {
+    compile('com.intel.gkl:gkl:0.4.3') {
```

Comment:
Can you rebase this branch onto the latest master before we review @erniebrau ? We are already on GKL 0.4.3 in this repo. Thanks!